src/test/kotlin/io/github/raeperd/realworldspringbootkotlin/ArticleIntegrationTest.kt

```diff
@@ -93,17 +95,40 @@ class ArticleIntegrationTest(
             .andExpect { status { isOk() } }
             .andReturnMultipleArticles()
             .apply { assertThat(articles.size).isEqualTo(articlesCount).isEqualTo(5) }
+
+        mockMvc.get("/articles?author=${author.username}")
+            .andExpect { status { isOk() } }
+            .andReturnMultipleArticles()
+            .apply { assertThat(articles.size).isEqualTo(articlesCount).isEqualTo(10) }
```

Comment:
This assertions repeats too many times.
Extract it to extension function of `MultipleArticleModel`